twigstan.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+return TwigStan\Config\TwigStanConfig::configure(__DIR__)
+    ->reportUnmatchedIgnoredErrors(true)
+    ->phpstanConfigurationFile(__DIR__.'/phpstan.neon')
+    ->phpstanMemoryLimit(false)
+    ->twigEnvironmentLoader(__DIR__.'/twig-loader.php')
+    ->twigPaths(__DIR__.'/templates')
```

Comment:
The default value for `twigPaths` was realy weid, where not putting the default Twig main directory tempate here? /cc @ruudk